hermes/bin/src/main.rs

```diff
@@ -2,17 +2,48 @@
 
 mod app;
 mod event;
+mod logger;
 mod reactor;
 mod runtime_context;
 mod runtime_extensions;
 mod wasm;
 
+use std::process;
+
+use tracing::{error, info, instrument};
 #[cfg(feature = "bench")]
 pub use wasm::module::bench::{
     module_hermes_component_bench, module_small_component_bench,
     module_small_component_full_pre_load_bench,
 };
 
+build_info::build_info!(fn build_info);
+
+// Disable process exit for clippy.
+#[allow(clippy::exit)]
+#[instrument]
 fn main() {
-    println!("Hello, world!");
+    // Initialize logger.
+    if let Err(err) = logger::init(logger::LogLevel::Info, true, true, true) {
```

Comment:
Get the log level from an env var, default to `info` if not set.
Don't worry about CLI arguments for now, just use an env var